ballot/app/scripts/deploy_service.js

```diff
@@ -1,7 +1,7 @@
 const chalk = require('chalk');
```

Comment:
This change should not be in this PR. It's included in https://github.com/oasislabs/tutorials/pull/49
Ditto for `hello_world`.

Reply:
addressed in https://github.com/oasislabs/tutorials/pull/55